spec/features/lender_portfolio_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.feature "lender portfolio page" do
+  let(:lender) { User.create(email: "example@example.com",
+                             password: "password",
+                             name: "richard",
+                             role: 0)
+  }
+
+  let!(:borrower) { User.create(email: "jeff@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.